padinfo/view/otherinfo.py

```diff
@@ -34,13 +34,16 @@ def statsbox(m, plus: int):
     tbl.hrules = prettytable.NONE
     tbl.vrules = prettytable.NONE
     tbl.align = "l"
-    levels = (m.level, 110, 120) if m.limit_mult > 0 else (m.level,)
+    levels = (1, m.level, 110, 120) if m.limit_mult > 0 else (m.level,)
     inh_tuple = (False, True) if plus == 297 else (False,)
     for lv in levels:
         for inh in inh_tuple:
             hp, atk, rcv, _ = m.stats(lv, plus=plus, inherit=inh)
             row_name = '(Inh)' if inh else 'Lv{}'.format(lv)
-            tbl.add_row([row_name, hp, atk, rcv])
+            if lv == 1 and inh:
+                pass
```

Comment:
You can make this slightly simpler like:

```python
# Don't show the inherit line for level 1
if lv == 1 and inh:
    continue
tbl.add_row(....
```

^ also I'd add a comment explaining why this extra control flow logic is there